tests/AbstractTestCase.php

```diff
@@ -13,9 +15,51 @@ protected function assertVastXmlEquals($expectedXml, Document $actualVastDocumen
         $actualXml = str_replace(
             array("\r", "\n"),
             '',
-            (string)$actualVastDocument
+            (string) $actualVastDocument
```

Comment:
https://github.com/php-fig-rectified/fig-rectified-standards/blob/master/PSR-2-R-coding-style-guide-additions.md#casting

Check PSR standarts. Use composer check-style to check if your code PSR-2 compatible.

Reply:
HM, I was drunk when write this comment) it's not from PSR.

Reply:
So, what's your final decision here? I still have a hope that I will send enough PRs to participate in Hacktoberfest :))))

Reply:
Currencly i use no spaces between cast and var. This style is preferred.